recipe/meta.yaml

```diff
@@ -13,11 +13,10 @@ source:
   sha256: a9e3acc58d8d67a08dbe6c86513b6bbff734a608a7fceb102883a88877643059
 
 build:
-  skip: true  # [python_impl == 'pypy']
```

Comment:
As [conda-forge has dropped PyPy]( https://conda-forge.org/news/2024/08/14/sunsetting-pypy/ ), this `skip` can also be dropped